packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -411,6 +449,27 @@ export function makeVirtualObjectManager(
     }
   }
 
+  function copyMethods(behaviorTemplate) {
+    const obj = {};
+    for (const [name, func] of Object.entries(behaviorTemplate)) {
+      assert.typeof(func, 'function');
+      obj[name] = func;
+    }
+    return obj;
+  }
+
+  function bindMethods(context, behaviorTemplate) {
+    const obj = {};
+    for (const [name, func] of Object.entries(behaviorTemplate)) {
+      assert.typeof(func, 'function');
+      const method = (...args) => Reflect.apply(func, null, [context, ...args]);
```

Comment:
I'll definitely defer to @erights here. But does this mean we're calling `Reflect.apply` every time someone invokes the method? I would have assumed that there was a way to bind the `this` and the first N arguments, but return a single (stable) function that could be called with additional arguments at runtime. Might be better for performance?

Reply:
We created our own `uncurryThis`, but it just calls `Reflect.apply` to solve the underlying problem.

Reply:
This is the incantation that @erights gave me.  I have no investment in it.  If a better recipe comes along, I'm happy to use it.  In the meantime I think this is the best we can do.